shared/src/components/pinned-header-footer-card/index.cjsx

```diff
@@ -46,7 +46,7 @@ module.exports = React.createClass
     @setBodyClassList()
 
   componentWillUnmount: ->
-    document.body.className = without(document.body.classList, @desiredBodyClassList()).join(' ')
+    document.body.className = @state.originalClassName
 
```

Comment:
hm, I think what's going on here is that if there's two of these on the screen they:

comp A mounts, remembers that body has "C,D" class, it adds E

comp B mounts, sees body has "C,D,E", remembers that list, and add F.

Now comp A unmount, and removes "C,D, E", so body now has "F".

comp B unmount, and sets it back to what it remembered as the original, which is: "C,D,E"

I think the only way to work around this is for the component to remove only things it adds, and forget about attempting to "remember" the original.